subsys/bluetooth/mesh/light_temp_srv.c

```diff
@@ -234,12 +234,10 @@ static void lvl_delta_set(struct bt_mesh_lvl_srv *lvl_srv,
 
 	bt_mesh_light_temp_srv_set(srv, ctx, &set, &status);
 
-	/* Override last temp value to be able to make corrective deltas when
-	 * new_transaction is false. Note that the last temp value in
-	 * persistent storage will still be the target value, allowing us to
-	 * recover correctly on power loss.
+	/* Copy start level to be able to make corrective deltas when
+	 * new_transaction is false.
 	 */
-	srv->transient.last.temp = lvl_to_temp(srv, start_lvl);
+	srv->corrective_delta = lvl_to_temp(srv, start_lvl);
```

Comment:
When `currective_delta` is stored?

Reply:
Not sure if I got the question. Could you elaborate?

Reply:
`transient.last.temp` is stored in flash. But when transition happens with delta set message, the temporary value is used. It should eventually be stored, shouldn't it?

Reply:
Hmm... as far as I can see persistent storage of `srv->transient.last.temp` is triggered by `bt_mesh_light_temp_srv_set`. According to the previous comment it did not seem like it was the intention to store the corrective delta in persistent storage. I belive it was an attempt to piggyback on the parameter to save memory, but it failed due to delayed persistent storage. Am I missing something essential here?

Reply:
```
/* Override last temp value to be able to make corrective deltas when
	 * new_transaction is false. Note that the last temp value in
	 * persistent storage will still be the target value, allowing us to
	 * recover correctly on power loss.
```